### DISCUSSION TITLE
有没有可能出一个脚本，可以让paopaodns在PVE的lxc中直接部署，而不是部署在lxc中的docker里

### DISCUSSION THREAD

**Discussion by machao1999** [kkkgo/PaoPaoDNS#97] [想法]
如题，现在使用pve做allinone，功能一切正常。但是总觉得pve套lxc，再套docker效率会低。现在paopaodns的docker是用的macvlan网桥再通过pve的内部网桥直连ros，理论上没有经过物理口。这样做和直接在lxc中部署paopaodns，有多少性能损失。

**Comment by kkkgo** [comment]
docker是基于内核隔离，理论上不会有多少性能损失，或者说实际场景不足以消耗大量的性能。
经过docker再nat一次的话网络确实会变差，macvlan模式可以减少一层nat。如果没有其他端口服务冲突，也可以直接用host模式（直接使用宿主的网络接口）。

**Comment by coLATin** [comment]
赞同。
